mis_builder/models/mis_report_instance.py

```diff
@@ -940,3 +963,15 @@ def _get_drilldown_action_name(self, arg):
             return f"{kpi.description} - {account.display_name} - {period.display_name}"
         else:
             return f"{kpi.description} - {period.display_name}"
+
+    def _get_annotation_context(self):
+        """
+        Return context of annotation linked to this instance
+        """
```

Comment:
```suggestion
        """Return the context used to filter annotation linked to this instance."""
```

Also, nit: https://peps.python.org/pep-0257/